### ISSUE TITLE
Procedure for monthly routine maintenance

### ISSUE THREAD

**Issue by FooBarWidget** [fullstaq-labs/fullstaq-ruby-infra#6]
We should document how to do routine maintenance. I can think of a few things to do:

 - Check whether we are using Kubernetes alpha/beta API versions, and upgrade them if so.
 - Check whether the Kubernetes cluster has an upgrade available, and upgrade if so.
 - Check whether the Nginx minor version is still supported, and upgrade it if not.

**Comment by abtreece** [comment]
Recommending closure — this issue is scoped entirely to infrastructure that no longer exists.

The maintenance items listed (Kubernetes API version upgrades, Kubernetes cluster upgrades, Nginx version upgrades) are all moot after the July 2024 revamp replaced both Kubernetes and Nginx with a Hetzner VM running Caddy.

If a routine maintenance procedure is still desired, a new issue with updated scope (Caddy updates, OS patching, Terraform provider upgrades, etc.) would be more appropriate.

**Comment by FooBarWidget** [comment]
Indeed. We already have unattended-upgrades, and Terraform providers are unlikely to cause issues. So that only leaves Caddy: we have no auto-update mechanism there.
